server/services/userService.js

```diff
@@ -25,6 +25,7 @@ module.exports = (nomisClientBuilder) => {
       isPrisonUser,
       activeCaseLoad,
       activeCaseLoadId,
+      activeCaseLoads,
```

Comment:
can we just call this caseloads? 

`user.caseLoads` provides enough context - and will make the code easier to read in the pug template

Reply:
yes